platforms/android/library/src/androidTest/res/layout/activity_test.xml

```diff
@@ -18,7 +18,8 @@
         <io.element.android.wysiwyg.EditorEditText
             android:id="@id/rich_text_edit_text"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content" />
+            android:layout_height="wrap_content"
+            android:inputType="text|textNoSuggestions" />
```

Comment:
This fixes the `EditorEditTextInputTests.testHardwareKeyboardTyping` test: it seems like the issue was the IME's autocorrect feature getting in the way.